app/views/layouts/autotune/application.html.erb

```diff
@@ -10,6 +10,15 @@
   <title>Autotune</title>
 
   <%=stylesheet_link_tag 'autotune/application', media: 'all' %>
+
+  <%=javascript_include_tag 'i18n' %>
+  <%=javascript_include_tag 'translations' %>
+  <script type="text/javascript">
+    I18n.defaultLocale = "<%= I18n.default_locale %>";
+    I18n.locale = "<%= I18n.locale %>";
+    I18n.fallbacks = true;
+  </script>
```

Comment:
So I think this can work for now... but all other javascripts are being handled by node and browserify, so it's possible that this will break our javascript testing rig. There is very little test coverage in that rig currently which is why this will work for now, but it will have to be refactored in the future if we want to add more javascript tests.